includes/data.php

```diff
@@ -1729,6 +1723,15 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 
 	$original_value = $value;
 
+	if ( null === $params->and ) {
+		$params->and = ' ' . __( 'and', 'pods' ) . ' ';
+	}
+
+	$params->and       = apply_filters( 'pods_serial_comma_and', $params->and, $value, $original_value, $params );
```

Comment:
```suggestion
	/**
	 * Allow filtering the "and" content used for pods_serial_comma.
	 *
	 * @since 2.7.17
	 *
	 * @param string|null $and The "and" content used, return null to disable.
	 * @param string $value    The value input into pods_serial_comma.
	 * @param object $params   The list of the setup parameters for pods_serial_comma.
	 */
	$params->and = apply_filters( 'pods_serial_comma_and', $params->and, $value, $params );
```